hal/transport/RFM69/driver/new/RFM69_new.cpp

```diff
@@ -384,7 +443,16 @@ LOCAL bool RFM69_sendFrame(rfm69_packet_t *packet, const bool increaseSequenceCo
 	while (!RFM69_channelFree() &&
 	        ((hwMillis() - CSMA_START_MS) < MY_RFM69_CSMA_TIMEOUT_MS)) {
 		doYield();
+		
+		//A little delay before the next attempt
+		delay(1);
+	}
+	
```

Comment:
Earlier behavior was to send anyway if the CSMA time limit was reached. If the new behavior is used, messages will not be sent at all in environments like mine (using the default settings). See https://forum.mysensors.org/topic/10224/csma-lbt-for-rfm69-new-driver/
Not sure what is best here. I really want the library to work by default for most users. But sending if there is a lot of noise isn't nice, so in that aspect the new behavior is better.

Reply:
If we opt for blocking the message, there needs to be a very clear log message (visible on normal debug level or even higher if possiblej. Otherwise nodes will just silently drop all outgoing messages and people will be very confused.